### PR TITLE
feat(simple_planning_sim): publish sensing interface imu data

### PR DESCRIPTION
**Pull request by TakaHoribe**
## Description

Add imu publisher in the `simple_planning_simulator` so that the planning modules using imu info can run.

## Tests performed

Run psim and check if the imu topic is published:
```
ros2 topic echo /sensing/imu/imu_data
```

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].
- [x] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.
- [x] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by purewater0901 — APPROVED** [review]
LGTM

**Comment by purewater0901** [comment]
@TakaHoribe It works well on PSim

File: simulator/simple_planning_simulator/src/simple_planning_simulator/simple_planning_simulator_core.cpp [hunk 1]
```diff
@@ -563,6 +565,27 @@ void SimplePlanningSimulator::publish_acceleration()
   pub_acc_->publish(msg);
 }
 
+void SimplePlanningSimulator::publish_imu()
+{
+  sensor_msgs::msg::Imu imu;
+  imu.header.frame_id = "/base_link";
```
Comment (tkimura4) [inline simulator/simple_planning_simulator/src/simple_planning_simulator/simple_planning_simulator_core.cpp:571]: /base_link -> base_link ?

File: simulator/simple_planning_simulator/src/simple_planning_simulator/simple_planning_simulator_core.cpp [hunk 2]
```diff
@@ -563,6 +565,27 @@ void SimplePlanningSimulator::publish_acceleration()
...
+  imu.linear_acceleration_covariance.at(3) = COV;
+  imu.linear_acceleration_covariance.at(6) = COV;
+  imu.angular_velocity = current_odometry_.twist.twist.angular;
+  imu.angular_velocity_covariance.at(0) = COV;
+  imu.angular_velocity_covariance.at(3) = COV;
+  imu.angular_velocity_covariance.at(6) = COV;
+  imu.orientation = current_odometry_.pose.pose.orientation;
+  imu.orientation_covariance.at(0) = COV;
+  imu.orientation_covariance.at(3) = COV;
+  imu.orientation_covariance.at(6) = COV;
```
Comment (tkimura4) [inline simulator/simple_planning_simulator/src/simple_planning_simulator/simple_planning_simulator_core.cpp:585]: 0, 3, 6 -> 0, 4, 8?

Comment (tkimura4) [reply simulator/simple_planning_simulator/src/simple_planning_simulator/simple_planning_simulator_core.cpp:585]: https://github.com/autowarefoundation/autoware.universe/blob/main/sensing/imu_corrector/src/imu_corrector_core.cpp#L81-L92
https://github.com/autowarefoundation/autoware.universe/blob/main/common/tier4_autoware_utils/include/tier4_autoware_utils/ros/msg_covariance.hpp#L26-L34

File: simulator/simple_planning_simulator/src/simple_planning_simulator/simple_planning_simulator_core.cpp (hunk 1 above)
Comment (TakaHoribe) [reply simulator/simple_planning_simulator/src/simple_planning_simulator/simple_planning_simulator_core.cpp]: addressed in https://github.com/autowarefoundation/autoware.universe/pull/2843/commits/8bb8b70a55e4edaa791683ed407043c8c9c11d38

File: simulator/simple_planning_simulator/src/simple_planning_simulator/simple_planning_simulator_core.cpp (hunk 2 above)
Comment (TakaHoribe) [reply simulator/simple_planning_simulator/src/simple_planning_simulator/simple_planning_simulator_core.cpp]: addressed in https://github.com/autowarefoundation/autoware.universe/pull/2843/commits/8bb8b70a55e4edaa791683ed407043c8c9c11d38

**Review by tkimura4 — APPROVED** [review]
LGTM

**Pull request merged**